### PR TITLE
Commiting jet html directory, tested compilation

### PR DESCRIPTION
**Pull request by jamesjl3**
Awaiting comments from Derek Anderson for this draft PR

### CODE REVIEW

File: subsystems/jets/JetDraw.h [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+  void SetJetSummary(TCanvas* c);
+ 
+  // void SetCemcChecker(JetGoodRunChecker* ch) {cemc_checker = ch;} TBD -- topical group input
+
+ private:
+  int MakeCanvas(const std::string &name, int num);
+  QADrawClient* cl; // Declare cl here   
+  std::vector<std::vector<TCanvas*>> m_vecCanvas;  // Declare m_vecCanvas
+  int nDrawError;
+  int idraw;
```
Comment (ruse-traveler) [inline subsystems/jets/JetDraw.h:46]: These should be local variables since they're only used in `Draw()`

File: subsystems/jets/JetDraw.cc [hunk 2]
```diff
@@ -0,0 +1,860 @@
...
+  int resolutionDrawError = 0;
+  for (uint32_t trigToDraw : m_vecTrigToDraw) {  // Loop over triggers
+    for (auto& resAndName : m_mapResToName) {  // Loop over resolution and name pairs
+      std::string pngFile = cl->htmlRegisterPage(
+						 *this,
+						 m_mapTrigToName[trigToDraw] + "/" + std::to_string(resAndName.first),
+						 m_mapTrigToName[trigToDraw] + "_" + resAndName.second,
+						 "png");
+      // Convert canvas to PNG
+      resolutionDrawError = cl->CanvasToPng(m_vecCanvas[trigToDraw][resAndName.first], pngFile);
```
Comment (ruse-traveler) [inline subsystems/jets/JetDraw.cc:70]: Correct me if I'm wrong, but I think as written nothing will actually be drawn on any of the canvases in `m_vecCanvas`? Rather, it looks like canvases get added to the vector, but everything actually gets drawn on the canvases in `TC`...

**Review by ruse-traveler — CHANGES_REQUESTED** [review]
This is almost there! The majority of my comments are just clean-up (removing unnecessary variables, comments, etc.), but I do think there is an issue with how the canvases are being handled...

Right now, both `TC` and `m_vecCanvas` are being used in different places, and we'll want to use one or the other. My preference is for vectors, but we can discuss!

File: subsystems/jets/JetDraw.cc (hunk 2 above)
Comment (jamesjl3) [reply subsystems/jets/JetDraw.cc:63]: Ah yes you're right. The actual drawing operations are not performed on the canvases stored in m_vecCanvas, but rather on separate canvases and then converted to PNG files using the cl->CanvasToPng() function. The purpose of the m_vecCanvas vector was to manage and store the canvases for each trigger and resolution, but the drawing happens via TC elsewhere

**Review by jamesjl3 — COMMENTED** [review]
using vectors instead of arrays for managing the canvases, Derek fixed a few minor things like white space (e.g. cleaned up cases where a bunch of spaces ended up at the end of lines), added missing histogram tags/characters (there were some underscores and "antikt_towersub1" tags missing), and made return codes consistent between the different methods ...

**Pull request merged**